src/rhsm/profile.py

```diff
@@ -45,12 +45,12 @@ def __init__(self, name, version, release, arch, epoch=0, vendor=None,
     def to_dict(self):
         """ Returns a dict representation of this packages info. """
         return {
-                'name': self.name,
-                'version': self.version,
-                'release': self.release,
-                'arch': self.arch,
-                'epoch': self.epoch,
-                'vendor': self._normalize_string(self.vendor),  # bz1519512 handle vendors that aren't utf-8
+                'name': self._normalize_string(self.name),
+                'version': self._normalize_string(self.version),
+                'release': self._normalize_string(self.release),
+                'arch': self._normalize_string(self.arch),
+                'epoch': self._normalize_string(self.epoch),
+                'vendor': self._normalize_string(self.vendor),
```

Comment:
Please, do not remove comment: `# bz1519512 handle vendors that aren't utf-8` and you can add similar comment to BZ bug report: 1543639